File: AD-HOC/2343/main.dart
import 'dart:collection';
import 'dart:io';

bool samePlace(int n) {
  HashSet<String> lightnings = new HashSet<String>();
  for (int i = 0; i < n; i++) lightnings.add(stdin.readLineSync()!);
  print(lightnings);
  return lightnings.length < n;
}

void main() {
  int n = int.parse(stdin.readLineSync()!);
  print(samePlace(n) ? '1' : '0');
}

File: AD-HOC/2343/main.js
function samePlace(n) {
    let lightnings = new Set();
    for (let i = 0; i < n; i++) lightnings.add(lines[i]);
    return lightnings.size < n;
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let n = parseInt(lines.shift());
console.log(samePlace(n) ? '1' : '0');
